scripts/GenerateUnicodeBlock.sc

```diff
@@ -0,0 +1,411 @@
+// Run with $ amm scripts/GenerateUnicodeBlock.sc
+//   Replace code in ./javalanglib/src/main/scala/java/lang/Character.scala
+
+import java.net.URL
+import java.nio.charset.StandardCharsets
+
+val blocksSourceURL: URL = new URL("http://unicode.org/Public/UCD/latest/ucd/Blocks.txt")
+
+// Generated/created via Mapping Worksheet:
+//   https://docs.google.com/spreadsheets/d/174aedhJlWNZYEEJtSVcIiMFvD7f27rg4NssjFw0hIDw/edit?usp=sharing
+
+val ucdBlockNameToJDKBlockName: Map[String, String] = Map(
+  "Adlam" -> "ADLAM",
```

Comment:
What's the point of a script if we need a huge list in the script itself anyway?

I would much rather have a script similar to the ones we already have in `Character`, which work by looping over all the possible code points and extracting results from the JVM itself. See for example the script of `Character.charTypeIndices`.